addons/l10n_nl_edi/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Netherlands - E-Invoicing (NLCIUS 1.0.3)',
+    'version': '0.1',
+    'category': 'Accounting/Localizations/EDI',
+    'summary': 'E-Invoicing, Universal Business Language (NLCIUS 1.0.3), nlc',
+    'description': """
+NLCIUS is the Dutch implementation of EN 16931 norm (with some limitations). Both for UBL and UN / CEFACT XML CII.
```

Comment:
I would not mind a little description about how the NLCIUS is the NL dialect of the BIS 3.0 and if it can be sent standalone or you need the PEPPOL network for it.